website/tsconfig.json

```diff
@@ -12,5 +12,5 @@
     },
     "types": ["ember-source/types"]
   },
-  "include": ["app/**/*", "tests/**/*", "types/**/*"]
+  "include": ["app/**/*", "tests/**/*", "types/**/*", "docs/**/*"]
```

Comment:
should we be more specific with the glob and point to `ts/gts` files?

Reply:
no I dont think so, we dont do that in any other tsconfig files in the monorepo.